Static template fails to deploy to Netlify due lack of `build` command

Steps to reproduce:
1. Create new sandbox using "Static" template
2. Deploy to Netlify from CodeSandbox left sidebar
3. Wait until build kicks off
4. Click "View logs" (after it appears under the Deploy button)

Expected: Site deploys to Netlify
Actual: `npm ERR! missing script: build`

Reproduction: https://codesandbox.io/s/silly-https-4isim

--- a/package.json
+++ b/package.json
@@ -4,7 +4,8 @@
   "description": "This is a static template with no bundling",
   "main": "index.html",
   "scripts": {
-    "start": "serve"
+    "start": "serve",
+    "build": "echo This is a static template, there is no bundler or bundling involved!"
   },
   "repository": {
     "type": "git",

